server/routes/itemImages.js

```diff
@@ -46,7 +44,7 @@ router.route('/upload/:itemId').post(singleUpload, registeredUser, ownershipGuar
 });
 
 // post image LINKS (not files)
-router.route('/link/:itemId').post(registeredUser, ownershipGuard, (req, res) => {
+router.route('/link/:itemId').post((req, res) => {
```

Comment:
include loggedin guard here
post( -- must be logged in --, singleUpload, (req, res) => {